data-index/data-index-storage/data-index-storage-infinispan/src/main/java/org/kie/kogito/index/infinispan/schema/ProtoSchemaManager.java

```diff
@@ -109,7 +109,7 @@ private String getTemplateRendered(SchemaDescriptor schemaDescriptor, String cac
 
     private void logProtoCacheKeys() {
         LOGGER.debug(">>>>>>list cache keys start");
-        protobufCacheService.getProtobufCache().entrySet().forEach(e -> LOGGER.debug(e.toString()));
+        protobufCacheService.getProtobufCache().entries().values().forEach(e -> LOGGER.debug(e));
```

Comment:
As the method name suggests, keys should be listed here.
```suggestion
        protobufCacheService.getProtobufCache().entries().keySet().forEach(e -> LOGGER.debug(e));
```

Reply:
@xieshenzh thanks, I reverted to log as it was before.